src/nvme/linux.c

```diff
@@ -795,6 +795,39 @@ long nvme_lookup_keyring(const char *keyring)
 	return keyring_id;
 }
 
+char *nvme_describe_key_serial(long key_id)
+{
+	char *desc;
+
+	if (keyctl_describe_alloc(key_id, &desc) < 0)
+		desc = NULL;
+	return desc;
+}
+
+long nvme_lookup_key(const char *type, const char *identity)
+{
+	key_serial_t key;
+
+	key = keyctl_search(KEY_SPEC_SESSION_KEYRING, type, identity, 0);
+	if (key < 0) {
+		nvme_msg(NULL, LOG_ERR,
```

Comment:
`keyctl_search` seems to set `errno`, I would just document this and don't print anything.

Reply:
Sure.

Reply:
Done with the update.